packages/playground/src/dashboard/components/set_extra_fee.vue

```diff
@@ -33,6 +33,7 @@
                   validators.required('Fee is required.'),
                   validators.isNumeric('Fee must be a valid number.'),
                   validators.min('Fee must be a 0 or more.', 0),
+                  validators.max('Maximum allowed fee is 1000000000000000', 1000000000000000),
```

Comment:
this limit is too high for the node's extra fee, let's limit it to 1 million or less

Reply:
should be resolved